L1Trigger/L1TTrackMatch/plugins/L1TrackJetClustering.h

```diff
@@ -17,6 +17,22 @@ namespace l1ttrackjet {
   const unsigned int ETA_INTPART_BITS{3};
   const unsigned int kExtraGlobalPhiBit{4};
 
+  static constexpr int kEtaWordLength = 15;
+  static constexpr int kPhiWordLength = 12;
+
+  //Constants used for jet clustering in eta direction
+  static constexpr int kThirteenBitMask = 8191;  // 0001111111111111 (bin) = 8191 (dec)
```

Comment:
Could these be made into binary literals? Then you don't need the comment and it's clearer for all of the cases.